misc/extensions.cfg

```
@@ -0,0 +1,2 @@
+[server]
+enabled_extensions=ui
```

Config options are case insensitive, but the documentation pages use a `-` everywhere, so it would be good to use a `-` here as well.

```suggestion
enabled-extensions=ui
```